From PeterHuewe@gmx.de  Mon Oct 26 14:31:48 2009
From: Peter Huewe <PeterHuewe@gmx.de>
Date: Tue, 29 Sep 2009 01:40:59 +0200
Subject: Staging: cx25821:  adding __init/__exit macros
To: "Greg Kroah-Hartman" <gregkh@suse.de>
Cc: Mauro Carvalho Chehab <mchehab@redhat.com>, Palash Bandyopadhyay <palash.bandyopadhyay@conexant.com>, devel@driverdev.osuosl.org, linux-kernel@vger.kernel.org, Jiri Kosina <trivial@kernel.org>, kernel-janitors@vger.kernel.org
Message-ID: <200909290141.00324.PeterHuewe@gmx.de>
Content-Disposition: inline


From: Peter Huewe <peterhuewe@gmx.de>

Trivial patch which adds the __init/__exit macros to the module_init/
module_exit functions of ./staging/cx25821/cx25821-core.c

Signed-off-by: Peter Huewe <peterhuewe@gmx.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/cx25821/cx25821-core.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/staging/cx25821/cx25821-core.c
+++ b/drivers/staging/cx25821/cx25821-core.c
@@ -1521,7 +1521,7 @@ static struct pci_driver cx25821_pci_dri
 	.resume = NULL,
 };
 
-static int cx25821_init(void)
+static int __init cx25821_init(void)
 {
 	INIT_LIST_HEAD(&cx25821_devlist);
 	printk(KERN_INFO "cx25821 driver version %d.%d.%d loaded\n",
@@ -1530,7 +1530,7 @@ static int cx25821_init(void)
 	return pci_register_driver(&cx25821_pci_driver);
 }
 
-static void cx25821_fini(void)
+static void __exit cx25821_fini(void)
 {
 	pci_unregister_driver(&cx25821_pci_driver);
 }
